### PR TITLE
Add new Go framework: Fiber

### PR DESCRIPTION
**Pull request by Fenny**
Hello,

I would like to add [Fiber](https://github.com/fenny/fiber) to the web-frameworks.
Local benchmark passed all tests, let me know if I'm missing something.

Thank you!

### CODE REVIEW

File: go/fiber/go.mod [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module main
```
Comment (waghanza) [inline go/fiber/go.mod]: could you reduce this file using 
```
module main

require github.com/fenny/fiber v0.4.0
```
(without specifying `go` **version**)

it reduce maintenance stuff :stuck_out_tongue:

File: go/fiber/config.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: github.com/fenny/fiber
```
Comment (waghanza) [inline go/fiber/config.yaml]: rather use `website: fenny.github.io/fiber/#/` no ? to link directly the documentation from `here`

or `github: fenny/fiber` if you prefer to link github directly

**Comment by Fenny** [comment]
Fixed both, thanks for your kind words!

**Comment by Fenny** [comment]
Missed it, my bad!

**Comment by waghanza** [comment]
Hi @Fenny,

This `PR` is now valid :+1: 

However, I'll not merge as-is. As this `PR` is made from **master** (unwritable by me), I'll open a new `PR` (with your commits) where I will update results to include `fiber` :heart_eyes: 

Regards,

**Comment by Fenny** [comment]
@waghanza 

Thank you +1, can't wait to see some results :dancing_women:

**Comment by waghanza** [comment]
@Fenny patience ... It took me ~6 hours to compute results :stuck_out_tongue_closed_eyes: 

However, be aware that results are computed on **local** `docker`, so might not be as **real-word** as expected.

I am working on a `cloud` version (in few weeks) to be closer as **real-world**

**Comment by waghanza** [comment]
See https://github.com/the-benchmarker/web-frameworks/issues/1618#issue-474964843 for the roadmap

**Comment by Fenny** [comment]
If you could merge my pull request, I can run this repo on a dedicated server and post results :+1:

**Comment by waghanza** [comment]
@Fenny missed you comment

I prefer no to do that, I will be awesome to have more infrastructure, but I expect to :

1. run on `cloud`
2. store results on a `cloud` database
3. have a `graphql` engine to get data
4. have a blog that display beautiful graphs about those results

after step 2, I will be more than happy to have results from multiple infrastructure stored :heart:

results will be available in ~2 hours :stuck_out_tongue:

**Comment by Fenny** [comment]
@waghanza I will stay on the look-out :+1:

**Pull request closed without merging**